src/shared/GlobalBanners/SelfHostedLicenseExpiration/LicenseExpirationModal/LicenseExpirationModal.tsx

```diff
@@ -0,0 +1,94 @@
+import A from 'ui/A'
+import Button from 'ui/Button'
+import Modal from 'ui/Modal'
+
+interface LicenseExpirationModalArgs {
+  isModalOpen: boolean
+  isLicenseExpired: boolean
+  isSeatsLimitReached: boolean
+  setIsModalOpen: (x: boolean) => void
+  isLicenseExpiringWithin30Days: boolean
+}
+
+const LicenseExpirationModal = ({
+  isModalOpen,
+  setIsModalOpen,
+  isSeatsLimitReached,
+  isLicenseExpired,
+  isLicenseExpiringWithin30Days,
+}: LicenseExpirationModalArgs) => {
```

Comment:
I would suggest swapping this over to:

```tsx
const LicenseExpirationModal: React.FC<LicenseExpirationModalArgs> = ({ /* ... */ }) => {
  // ...
}
```

That way you get the correct return type checks